src/sentry/static/sentry/app/views/settings/organizationTeams/teamMembers.tsx

```diff
@@ -88,24 +100,21 @@ class TeamMembers extends React.Component {
           });
           addSuccessMessage(t('Successfully removed member from team.'));
         },
-        error: () => {
+        error: () =>
```

Comment:
I think it's fine, but I would not return the `addErrorMessage`

Reply:
I think this is okay, it doesn't necessarily matter since the return is `void`